src/library/autodj/dlgautodj.h

```diff
@@ -23,13 +23,14 @@ class DlgAutoDJ : public QWidget, public Ui::DlgAutoDJ, public LibraryView {
               Library* pLibrary,
               AutoDJProcessor* pProcessor, TrackCollection* pTrackCollection,
               KeyboardEventFilter* pKeyboard);
-    virtual ~DlgAutoDJ();
-
-    void onShow();
-    void onSearch(const QString& text);
-    void loadSelectedTrack();
-    void loadSelectedTrackToGroup(QString group, bool play);
-    void moveSelection(int delta);
+    virtual ~DlgAutoDJ() override;
```

Comment:
You can actually drop all the virtuals here (and in the other dialogs) -- override should be enough. We didn't need it before either, but we kind of used it as a hint that we were overriding a base class method but now override does that.